x-pack/plugins/observability_onboarding/e2e/cypress/support/commands.ts

```diff
@@ -96,6 +96,29 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add('installCustomIntegration', (integrationName: string) => {
+  const kibanaUrl = Cypress.env('KIBANA_URL');
+
+  cy.request({
+    log: false,
+    method: 'POST',
+    url: `${kibanaUrl}/api/fleet/epm/custom_integrations`,
+    body: {
+      force: true,
+      integrationName,
+      datasets: [
+        { name: 'access', type: 'logs' },
+        { name: 'error', type: 'metrics' },
+        { name: 'warning', type: 'logs' },
+      ],
+    },
+    headers: {
+      'kbn-xsrf': 'e2e_test',
```

Comment:
This seems like an external endpoint. Do we not need the version header ?

Reply:
It's not complaining like the `DELETE` one
should we still add it?

Reply:
this is really strange reason being, its not prefixed with `/internal` which means this should complain. 

Looking at [Fleet PRs](https://github.com/elastic/kibana/pull/165823), looks like something is fishy here and they might have missed adding this. 

I even see a commit name `add versioning to cypress tasks that call fleet api`

Reply:
I'll go ahead and add the header in all the public api calls, I agree with you something is weird

Reply:
Addressed in [fd721bb](https://github.com/elastic/kibana/pull/165782/commits/fd721bbf183e72c0e930470ccb11f0a7a4eb8611).